gradle/validation/forbidden-apis/commons-io.commons-io.all.txt

```diff
@@ -0,0 +1,2 @@
+@defaultMessage Use org.apache.commons.io.file.PathUtils#deleteDirectory(java.nio.file.Path) instead
```

Comment:
Can you move the last few lines of defaults.all.txt to keep all the commons io stuff together?